samples/bluetooth/peripheral_hr/overlay-phy_coded.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_BT_AUTO_PHY_UPDATE=n
+CONFIG_BT_EXT_ADV=y
+
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
```

Comment:
Seems like we set `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191` every time we do `CONFIG_BT_EXT_ADV=y`. Should we modify the default value of `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX`?